ci/build/build-release.sh

```diff
@@ -136,10 +136,8 @@ EOF
   ) > "$VSCODE_OUT_PATH/product.json"
 
   # Use the package.json for the web/remote server.  It does not have the right
-  # version though so pull that from the main package.json.  Also remove keytar
-  # since the web does not rely on it and that removes the dependency on
-  # libsecret.
-  jq --slurp '.[0] * {version: .[1].version} | del(.dependencies.keytar)' \
+  # version though so pull that from the main package.json.
+  jq --slurp '.[0] * {version: .[1].version}' \
```

Comment:
Nice catch! How'd you figure this out? That log message when code-server starts up? IIRC

Reply:
Fortunately we still have libsecret as a dependency in our docs (as far as I could tell, hopefully I did not miss a spot) so nothing else changes